improver_tests/acceptance/test_regrid.py

```diff
@@ -38,6 +38,7 @@
 
 pytestmark = [pytest.mark.acc, acc.skip_if_kgo_missing]
 GLOBAL_UK_TITLE = "Global Model Forecast on UK 2 km Standard Grid"
+MOGREPS_G_UK_TITLE = "MOGREPS-G Model Forecast on UK 2 km Standard Grid"
```

Comment:
The input data for these tests were a bit weird, and they still are after these changes. The file `global_cutout_multi_realization` is actually on a ~10km grid (at 50 degrees latitude) suggesting it is actually gl_det data that's been duplicated into three realizations. This makes sense as it is being used in conjunction with `glm_landmask.nc` which is a gl_det landmask.

This all means that this is a bit of a mess as things stand. We've got a gl_det landmask, gl_ens data, with the latter now described as being on a 2km grid. Could you please make everything self consistent:
- gl_ens data on a ~20km grid titled as such.
- a gl_ens suitable land mask

Reply:
I've modified all the fields that are on a 10km grid to say so (and excluded the word "Standard", as it isn't). I agree that we need ensemble data that matches an ensemble land-mask, but I haven't attempted this yet. Might have to wait until next week.

Reply:
I've now modified the global ensemble input data to come from the global ensemble (including the landmask).